sites/platform/src/dedicated-environments/security-monitoring.md

```diff
@@ -0,0 +1,84 @@
+
+- [Project isolation](#project-isolation)
+- [Encryption](#encryption)
+- [Performance monitoring](#performance-monitoring)
+- [Security incident monitoring](#security-incident-monitoring)
+
+{{% description %}}
+
+### Project isolation 
+
+All Dedicated clusters are single-tenant. The [three hosts](https://docs.platform.sh/dedicated-gen-2/overview.html) are exclusively used by a single customer and each cluster is launched into its own isolated network (VPC on AWS, equivalent on other providers). The network is behind a firewall for incoming connections. Only ports 22 (SSH), 80 (HTTP), and 443 (HTTPS), and 2221 (SFTP) are opened to incoming traffic.
+
+There are no exceptions for this rule, so any incoming web service requests, ETL jobs, or otherwise need to transact over one of these protocols. Outgoing TCP traffic isn’t behind a firewall. Outgoing UDP traffic is disallowed. For containers to be allowed to connect to each other, the following requirement must be met:
+
+-   The containers must live in the same environment
+-   You need to define an explicit relationship between the containers in your [app configuration](https://docs.platform.sh/create-apps/app-reference/single-runtime-image.html#relationships)
+
+All Dedicated projects are isolated and their data is fully encrypted. Should a security breach occur, Platform.sh follows a strict [security incident handling** procedure](https://docs.platform.sh/dedicated-gen-3/security.html#security-incident-handling-procedure) to deal with the issue as promptly and efficiently as possible.
+
+### Encryption 
+
+| AWS | GCP | Azure |
+|--------|--------------------------|--------------------------------|
+| AWS EBS Volumes are encrypted on Platform.sh, which means Dedicated Gen 3 and Dedicated Gen 2 sites are fully encrypted. Keys are managed by the AWS Key Management Service. AWS automatically rotates these keys every three years. In some cases, temporary storage (such as swap) is stored on unencrypted local storage volumes.| By default, data is encrypted using [Microsoft-Managed Keys](https://learn.microsoft.com/en-us/compliance/assurance/assurance-encryption) for Azure Blobs, Tables, Files, and Queues. | Data is encrypted using default encryption at rest. |
+
+
+### Performance monitoring
+
+All of our Dedicated clusters are monitored 24/7 to ensure uptime and to measure server metrics such as available disk space, memory and disk usage, and several dozen other metrics that give us a complete picture of the health of your application’s infrastructure.
+
+A wide range of server metrics, including disk space, memory, and disk usage are continuously measured using in-house tools. These metrics provide a complete picture of the health of your application infrastructure. As soon as a metric goes out of bounds, Platform.sh Support and Operations teams are alerted. When an outage is detected, a Point in Time report is generated so Platform.sh Support can triage the cause of the outage.
```

Comment:
tis is essentially the same sentence as the last sentence. Can we combine the two to be more succinct ?